google-cloud-datastore/test/google/cloud/datastore_test.rb

```diff
@@ -19,7 +19,7 @@
   describe "#datastore" do
     it "calls out to Google::Cloud.datastore" do
       gcloud = Google::Cloud.new
-      stubbed_datastore = ->(project, keyfile, scope: nil, timeout: nil, client_config: nil) {
+      stubbed_datastore = ->(project, keyfile, scope: nil, timeout: nil, host: nil, client_config: nil) {
```

Comment:
Check that host is nil (same as the other fields)

Reply:
Done!